.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Add the .idea folder to your .gitignore. If you need assistance doing this, let me know. Other than this, what you have looks good.